org.kde.kid3.json

```diff
@@ -0,0 +1,100 @@
+
+        /* Required to load music metadata */
+        "--share=network",
+
+        /* Required for input-output */
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-music",
+        "--env=TMPDIR=/var/tmp",
+
+        /* For MPRIS */
+        "--own-name=org.mpris.MediaPlayer2.kid3"
+    ],
+
+    "cleanup": [
+        "/man",
+        "/share/man",
+        "/include",
+        "/lib/*.a",
+        "/lib/*.la",
+        /* For support in both GNOME and KDE, Kid3 has independent components for KDE and Qt
+           Since we build against the KDE runtime, we don't need the -Qt metadata */
+        "/share/**/*-qt.*",
+        "/share/icons/**/**/**/*-qt.*"
+    ],
+
+    "modules": [
+        {
+            "name": "chromaprint",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/acoustid/chromaprint/archive/v1.5.0.tar.gz",
+                    "sha256": "5c8e0d579cb3478900699110aa961c1552a422a18741cf67dd62136b1b877c7b"
+                }
+            ]
+        },
+        {
+            "name": "taglib",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DBUILD_SHARED_LIBS=ON",
+                "-DWITH_MP4=ON",
+                "-DWITH_ASF=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://taglib.org/releases/taglib-1.11.1.tar.gz",
+                    "sha256": "b6d1a5a610aae6ff39d93de5efd0fdc787aa9e9dc1e7026fa4c961b26563526b"
+                }
+            ]
+        },
+        {
+
+            "name": "kid3",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DWITH_APPS=KDE;Qt;CLI",
+                "-DWITH_ID3LIB=OFF",
+                "-DDOCBOOK_XSL_DIR=docbook-xsl"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kid3/3.8.3/kid3-3.8.3.tar.xz",
+                    "sha256": "3556e7a76d65baf8b4258b955f6c93d7467a02a0b588ff7249c8ab96bc128377"
+                },
+                {
+                    "type": "archive",
+                    "dest": "docbook-xsl",
+                    "url": "https://github.com/docbook/xslt10-stylesheets/releases/download/release/1.79.2/docbook-xsl-1.79.2.tar.gz",
+                    "sha256": "966188d7c05fc76eaca115a55893e643dd01a3486f6368733c9ad974fcee7a26"
+                }
+            ],
+            "post-install" :[
+                "sed -i 's|<\/component>|<content_rating type=\"oars-1.1\"\/>\\n<\/component>|g' \/app\/share\/metainfo\/org.kde.kid3.appdata.xml"
```

Comment:
upstream

Reply:
Issue created, now up to them to tackle it so we can use it in a future release:
https://bugs.kde.org/show_bug.cgi?id=421497

Reply:
It's been committed upstream.